admin_guide/selfprovisioned_projects.adoc

```diff
@@ -10,31 +10,107 @@
+to the "admin" role for that project. You can view the default template with the
+following command:
 
-If you want to change the way the shape of self-provisioned projects, you can change the template.  The API will substitute the following parameters into the template:
+----
+$ osc edit template/project-request -n openshift
+----
 
- .  PROJECT_NAME - name of the project
+.Default New Template
+====
+----
+apiVersion: v1beta3
+kind: Template
+metadata:
+  name: project-request
+  namespace: openshift
+objects:
+- apiVersion: v1beta1
+  displayName: ${PROJECT_DISPLAYNAME}
+  kind: Project
+  metadata:
+    annotations:
+      description: ${PROJECT_DESCRIPTION}
+      displayName: ${PROJECT_DISPLAYNAME}
+    creationTimestamp: null
+    name: ${PROJECT_NAME}
+  spec: {}
+  status: {}
+- apiVersion: v1beta1
+  groupNames: []
+  kind: RoleBinding
+  metadata:
+    creationTimestamp: null
+    name: admins
+    namespace: ${PROJECT_NAME}
+  roleRef:
+    name: admin
+  userNames:
+  - ${PROJECT_ADMIN_USER}
+parameters:
+- name: PROJECT_NAME <1>
+- name: PROJECT_DISPLAYNAME <2>
+- name: PROJECT_DESCRIPTION <3>
+- name: PROJECT_ADMIN_USER <4>
+----
 
- .  PROJECT_DISPLAYNAME = display name of the project (defaults to PROJECT_NAME)
+<1> The name of the project
+<2> The display name of the project. Defaults to *PROJECT_NAME*.
+<3> The description of the project. Defaults to *PROJECT_DISPLAYNAME*.
+<4> The name of the requesting user.
 
- .  PROJECT_DESCRIPTION = description of the project (defaults to PROJECT_DISPLAYNAME)
+====
+////
+The API substitutes the following parameters into the template:
 
- .  PROJECT_ADMIN_USER = name of the requesting user
+[cols="4,8",options="header"]
+|===
+|Parameter |Description
 
+|*PROJECT_NAME*
+|The name of the project
 
+|*PROJECT_DISPLAYNAME*
+|The display name of the project. Defaults to *PROJECT_NAME*.
 
-=== Permissions
-Access to this API is granted via the `self-provisioner` role and the `self-provisioners` cluster role binding.  By default, it is available to all authenticated users.  
+|*PROJECT_DESCRIPTION*
+|The description of the project. Defaults to *PROJECT_DISPLAYNAME*.
 
-=== Disabling auto-provisioning
-Deleting the `self-provisioners` cluster role binding will deny permissions for auto-provisioning new projects.  If you remove this permission, you should consider setting the `projectRequestMessage` option inside of the master-config.yaml file with instructions for how users can request a project.  That field is a string that will be presented to user in the webui and the cli when they attempt to self-provision a project.  Some common examples are:
+|*PROJECT_ADMIN_USER*
+|The name of the requesting user.
+|===
+////
 
- .  Call Bob at (555) 867-5309 to request a project
+You can create your own custom template by using the above as an example, and
+defining the desired template's location in the *_master-config.yaml_* file.
```

Comment:
After the earlier `osc edit` command is removed, it may be good (for context/flow) to tweak and move this sentence to before the example file.